pages/linux/systemctl.md

```diff
@@ -19,7 +19,7 @@
 
 `systemctl enable/disable {{unit}}`
 
-- Mask/Unmask a unit, prevent it to be started on bootup:
+- Mask/Unmask a unit to prohibit enablement and manual activation:
```

Comment:
I'm biased toward the original, simpler 'prevent'.  What do you think?
```suggestion
- Mask/unmask a unit to prevent enablement and manual activation:
```

Reply:
I get that the manpage uses 'prohibit' but IMO simpler is better, at least for tldr-pages.

Reply:
@aherst sorrry, did you apply this suggestion? I'm confused.

Reply:
Sorry, I'm new at this and learning as I go. I thought you had applied the change and was asking if I objected. I'll figure out the right thing to do when I have some time this afternoon. Thanks for you patience.

Reply:
Np @aherst.  No, I didn't apply the change.  Click "Commit suggestion" then "Commit changes" to commit the changes.

Reply:
(sorry about the testing below, I needed to test if tldr-bot was working because we had some server maintenance ;))

Reply:
Thanks for you help.